test/interactions/keyInteractionTests.ts

```diff
@@ -4,91 +4,62 @@ var assert = chai.assert;
+    var aCallback: (asd: number) => void;
+    var bCallback: (asd: number) => void;
+    var $target: JQuery;
+
+    beforeEach(() => {
+      aCode = 65;
+      bCode = 66;
+      component = new Plottable.Component();
+      svg = TestMethods.generateSVG(400, 400);
+      keyInteraction = new Plottable.Interactions.Key();
+      aCallback = () => aCallbackCalled = true;
+      bCallback = () => bCallbackCalled = true;
       component.renderTo(svg);
-
-      var keyInteraction = new Plottable.Interactions.Key();
-
-      var aCode = 65; // "a" key
-      var bCode = 66; // "b" key
-
-      var aCallbackCalled = false;
-      var aCallback = () => aCallbackCalled = true;
-      var bCallbackCalled = false;
-      var bCallback = () => bCallbackCalled = true;
-
-      keyInteraction.onKeyPress(aCode, aCallback);
-      keyInteraction.onKeyPress(bCode, bCallback);
-      keyInteraction.attachTo(component);
-
-      var $target = $(component.background().node());
-
-      TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
-      $target.simulate("keydown", { keyCode: aCode });
-      assert.isTrue(aCallbackCalled, "callback for \"a\" was called when \"a\" key was pressed");
-      assert.isFalse(bCallbackCalled, "callback for \"b\" was not called when \"a\" key was pressed");
-
-      aCallbackCalled = false;
-      $target.simulate("keydown", { keyCode: bCode });
-      assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when \"b\" key was pressed");
-      assert.isTrue(bCallbackCalled, "callback for \"b\" was called when \"b\" key was pressed");
-
-      TestMethods.triggerFakeMouseEvent("mouseout", component.background(), -100, -100);
+      $target = $(component.background().node());
       aCallbackCalled = false;
-      $target.simulate("keydown", { keyCode: aCode });
-      assert.isFalse(aCallbackCalled, "callback for \"a\" was not called when not moused over the Component");
-
-      svg.remove();
+      bCallbackCalled = false;
     });
 
-    it("appropriate keyCode is sent to the callback", () => {
-      var svg = TestMethods.generateSVG(400, 400);
-      var component = new Plottable.Component();
-      component.renderTo(svg);
-
-      var keyInteraction = new Plottable.Interactions.Key();
-
-      var bCode = 66; // "b" key
-
-      var bCallbackCalled = false;
-      var bCallback = (keyCode: number) => {
-        bCallbackCalled = true;
-        assert.strictEqual(keyCode, bCode, "keyCode 65(a) was sent to the callback");
-      };
-
-      keyInteraction.onKeyPress(bCode, bCallback);
-
-      keyInteraction.attachTo(component);
-
-      var $target = $(component.background().node());
-
-      TestMethods.triggerFakeMouseEvent("mouseover", component.background(), 100, 100);
-      $target.simulate("keydown", { keyCode: bCode });
-      assert.isTrue(bCallbackCalled, "callback for \"b\" was called when \"b\" key was pressed");
-
+    afterEach(() => {
       svg.remove();
```

Comment:
`svg.remove()` should not be placed in an `afterEach()` block.  This allows us to see the svg if the test fails.